# gazebo_ros_control_params.yaml
igor/gazebo_ros_control/pid_gains:
 L_wheel_joint: {p: 1, i: 0.0, d: 0.5}
 R_wheel_joint: {p: 1, i: 0.0, d: 0.5}
